test/docker_tests/configurations/test_all_docker_composes.sh

```
@@ -0,0 +1,61 @@
+        }
+    }
+    /^[^#]/ && !/^$/ {
+        exit
+    }
+    ' "$file" >> "$LOG_FILE"
+
+    echo "" >> "$LOG_FILE"
+}
+
+# Function to run docker-compose test
+run_test() {
```

Shellcheck (0.11.0)

[error] Couldn't parse this brace group. Fix to allow more checks.

(SC1073)